src/main/java/nextstep/step2/Validator.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.step2;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Validator {
+    private final int MAX_USER_NAME_LENGH = 5;
+
+    public int checkLadderHeight(int ladderHeight) {
+        try {
+            if(ladderHeight > 0) {
+                return ladderHeight;
+            }
+            throw new Exception("사다리의 높이는 1 이상이어야 합니다.");
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new IllegalArgumentException(e.getMessage());
+        }
```

Comment:
하는 역할에 비해, depth가 깊은 듯 합니다. 

아래처럼 사용하는 건 어떨까요???
```suggestion
        if(ladderHeight <= 0) {
            throw new IllegalArgumentException("사다리의 높이는 1 이상이어야 합니다.");
        }

        return ladderHeight;
```

checkUserName()도 위 예제를 참고하셔서 간결하게 만드셨으면 좋겠습니다:)